pkgs/top-level/python-packages.nix

```diff
@@ -8,6 +8,16 @@
 
 self: super: with self; {
 
+  bootstrap = lib.recurseIntoAttrs {
+    flit-core = toPythonModule (callPackage ../development/python-modules/bootstrap/flit-core { });
+    installer = toPythonModule (callPackage ../development/python-modules/bootstrap/installer {
+      inherit (bootstrap) flit-core;
+    });
+    build = toPythonModule (callPackage ../development/python-modules/bootstrap/build {
+      inherit (bootstrap) flit-core installer;
+    });
+  };
+
   bootstrapped-pip = toPythonModule (callPackage ../development/python-modules/bootstrapped-pip { });
```

Comment:
Not sure if we should move this also into bootstrap

Reply:
I like this for consistency, but I am hoping instead to leave this here and pursue deprecating this for Python 3. I just need to think of a good way to do it (docs, warnings, something else). Opinions appreciated.